src/selectors/entities/users.js

```diff
@@ -212,11 +212,32 @@ function sortAndInjectProfiles(profiles, profileSet, skipInactive = false) {
 
 export const getProfiles = createSelector(
     getUsers,
-    (profiles) => {
-        return sortAndInjectProfiles(profiles, PROFILE_SET_ALL);
+    (state, filters) => filters,
+    (profiles, filters) => {
+        return sortAndInjectProfiles(filterProfiles(profiles, filters), PROFILE_SET_ALL);
     }
 );
 
+function filterProfiles(profiles, filters) {
+    if (!filters || Object.keys(filters).length === 0) {
+        return profiles;
+    }
+    const filteredProfiles = {};
+
+    for (const [id, profile] of Object.entries(profiles)) {
+        if (filters.role && filters.role !== '') {
+            if (profile.roles && profile.roles.indexOf(filters.role) !== -1) {
+                filteredProfiles[id] = profile;
+            }
+        } else if (filters.inactive) {
+            if (profile.delete_at !== 0) {
+                filteredProfiles[id] = profile;
+            }
+        }
+    }
```

Comment:
This looks like it will only work for a single filter at a time. E.g. if I supply a filters of `{role: 'system_admin', inactive: true}` I won't get inactive sysadmins, I'll just get sysadmins. By default I'd expect that both filters I provide work

Reply:
I did it this way keeping in my mind the UI in the ticket which didn't allow multi select. 

But looking it as a generic selector method it makes sense to support multiple filters here, as the consumers of this selector will expect it. Will change now.. 👍